cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -176,9 +176,10 @@ data:
   # Labels to add to the CSI RBD Deployments and DaemonSets Pods.
   # ROOK_CSI_RBD_POD_LABELS: "key1=value1,key2=value2"
 
-  # (Optional) Ceph Provisioner NodeAffinity.
+  # (Optional) Ceph CSI provisioner NodeAffinity(applied to both CephFS and RBD provisioner).
```

Comment:
can we rename Ceph CSI as CephCSI is a project name? Take care of it in other places also.